base/services/w32time/w32time.spec

```diff
@@ -0,0 +1,2 @@
+18 stdcall W32TimeSyncNow(wstr long long)
+21 stdcall ServiceMain(long ptr)
```

Comment:
Missing newline at end of file.

Reply:
Yes, this is a duplicate file with one line commented out. but I have no idea how to move it to a library. I will try and add the newline.